airbyte-integrations/bases/source-acceptance-test/source_acceptance_test/plugin.py

```diff
@@ -78,6 +82,26 @@ def pytest_generate_tests(metafunc):
             metafunc.parametrize("inputs", test_inputs)
 
 
+def pytest_collection_modifyitems(config, items):
```

Comment:
What this for?

Reply:
This is exactly to resolve related issue. Here we get the prepared test items and decorate them with `@pytest.mark.timeout(value)` decorator with ether default value or with value specified in `acceptance-test-config.yml`